scrapscript.py

```diff
@@ -4423,7 +4423,8 @@ def runsource(self, source: str, filename: str = "<input>", symbol: str = "singl
                 self.env.update(result.env)
             else:
                 self.env["_"] = result
-            print(result)
```

Comment:
lgtm but mabe just put this indented under 4425; we already have non-EnvObject case